extensions/markdown-language-features/media/markdown.css

```diff
@@ -157,7 +157,7 @@ blockquote {
 }
 
 code {
-	font-family: var(--vscode-editor-font-family, Menlo, Monaco, Consolas, "Droid Sans Mono", "Courier New", monospace, "Droid Sans Fallback");
+	font-family: var(--vscode-editor-font-family, "SF Mono", Monaco, Menlo, Consolas, "Ubuntu Mono", "Liberation Mono", "DejaVu Sans Mono", "Courier New", monospace);
```

Comment:
@mjbvz please review this change, I tried to adjust this to be our standard monospace font we use in the workbench by combining the rules from our `monaco-monospace-font` into one